sections/performance/block-loop.md

```diff
@@ -0,0 +1,73 @@
+# Don't block the event loop
+
+<br/><br/>
+
+Node handles the Event Loop mostly on a single thread rotating through multiple queues. Operations with high complexity, large json parsing, unsafe regex querieies, and large IO operations are some of the operations that can cause the Event Loop to stall. Avoid this off-loading CPU intensive tasks to a dedicated service (e.g. job server), or breaking long tasks into small steps then using the Worker Pool are some examples of how to avoid blocking the Event Loop.
+
+## Example
```

Comment:
Formatting issue - in all other pages, the title contains the main message, e.g. "Example: blocking the event loop"

Reply:
I think I've fixed this. When I view online or in VScode it looks normal.